docs/settingUp.md

```diff
@@ -63,9 +63,11 @@ More information can be found at [this documentation](https://help.github.com/ar
       Append a **same** unique id (e.g your name) to **each** of the default accounts found at the bottom of this file,
       e.g change `test.student1.account=alice.tmms` to `test.student1.account=alice.tmms.KevinChan`.
 
-## Step 4: Set up Eclipse IDE
+## Step 4: Set up an IDE
 
-Eclipse IDE is our preferred development environment. Support requests related to other IDEs will not be entertained.
+[Eclipse IDE](#eclipse) is our preferred development environment and will get full support. [IntelliJ IDEA](#intellij-idea) support is currently experimental and will receive limited support. Support requests related to other IDEs will not be entertained.
```

Comment:
Sorry, missed this one: for the second sentence, "support" is mentioned twice which is kind of weird. "IntelliJ IDEA ~~support~~ usage is currently experimental and will receive limited usage" sounds better.